CorsaSchoolManagementSystem/studentFees.php

```diff
@@ -0,0 +1,29 @@
+<?php 
+
+require('Core/init.php');
+
+if($_SESSION['loggedIn'] && $_SESSION['userType'] === UserType::ADMIN){
+    if($_POST['addfees']){
+        //student fees
+        $admin = new Admin(new Database());
+        $studentFees = new studentFees();
```

Comment:
Where from the studentFees class? Was it created earlier, if not you need to create it?